docs/site/migration/auth/authentication.md

```diff
@@ -0,0 +1,293 @@
+---
+lang: en
+title: 'Migrating built-in authentication'
+keywords: LoopBack 4.0, LoopBack 4, LoopBack 3, Migration
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/migration-authentication.html
+---
+
+This is a guide to describe how to migrate LoopBack 3 built-in User/AccessToken
+model based authentication.
+
+As the pre-work of this guide, we created an application completely migrated
+from
+[`loopback-example-access-control`](https://github.com/strongloop/loopback-example-access-control).
```

Comment:
I think we can just point that example repo as an example. Since this is a documentation (not a blog), we don't need to mention the example repo is a pre-work or not. :)

Reply:
rephrased it to
> There is an application completely migrated
from the LoopBack 3's access control example
[`loopback-example-access-control`](https://github.com/strongloop/loopback-example-access-control).